### PR TITLE
Production deployment uses nginx and gunicorn to serve static data.

### PR DESCRIPTION
**Pull request by govza**
Production deployment uses nginx and gunicorn to serve static data.

Added nginx conatiner,
gunicorn dependencies,
Updated launch command to auto collect static, make migrations and run gunicorn.
Nginx config, prevents timeout within 10min for /admin to manage recitations

[Tested](https://github.com/it-muslim/quran-words/wiki/Tests) with full reciter packages addition.

### CODE REVIEW

File: docker-compose-win.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# before run use command `docker volume create postgres`
+# run docker-compose -f docker-compose-win.yml up
+version: "3"
+services:
+  web:
+    build: .
+    container_name: web01
+    env_file:
+      - ./.env
+    command: >
```
Comment (al-muammar) [inline docker-compose-win.yml]: Create an `entrypoint.sh` and put it there.

File: docker-compose-win.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# before run use command `docker volume create postgres`
+# run docker-compose -f docker-compose-win.yml up
+version: "3"
+services:
+  web:
+    build: .
+    container_name: web01
```
Comment (al-muammar) [inline docker-compose-win.yml]: Why do you add `01` suffix to everything?

File: docker-compose-win.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    env_file:
+      - ./.env
+    command: >
+            sh -c "python3 manage.py collectstatic --no-input &&
+            python3 manage.py makemigrations &&
+            python3 manage.py migrate &&
+            gunicorn config.wsgi -b 0.0.0.0:8000 --workers 3 --timeout 600"
+    volumes:
+      - .:/www
+    ports:
```
Comment (al-muammar) [inline docker-compose-win.yml]: Remove it, you don't need to open this port. At most you need to expose it.
The same everywhere except nginx.

**Review by al-muammar — CHANGES_REQUESTED** [review]
Why do you need two `docker-compose` files?
Otherwise, looks pretty decent.

**Comment by govza** [comment]
docker-compose-win used to postgresql [bug workaround](https://github.com/docker/for-win/issues/445) for windows, it creates external volume. 
For linux it's not needed,  postgresql data is linked to volume /postgres/

File: docker-compose-win.yml (hunk 2 above)
Comment (govza) [reply docker-compose-win.yml]: It's one of our futures swarm :)

File: entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+python3 manage.py collectstatic --no-input
+python3 manage.py makemigrations
+python3 manage.py migrate
+gunicorn config.wsgi -b 0.0.0.0:8000 --workers 3 --timeout 600
```
Comment (al-muammar) [inline entrypoint.sh]: Try the following thing and then specify your command in `docker-compose`. In this way you don't need two entry points for development and production.
```suggestion
exec "$@"
```

**Review by al-muammar — APPROVED** [review]

**Review by Imangazaliev — APPROVED** [review]

**Pull request merged**